doc/remediation-flow.md

```diff
@@ -139,7 +123,7 @@ spec:
 ### The scan-remediate-repeat flow
 The general flow is common for the cluster scan as well as the node
```

Comment:
we should say Platform scans on this line too.

Reply:
fixed